### PR TITLE
refactor: simplify activities tab

### PR DESCRIPTION
**Pull request by danielpeintner**
proposal for https://github.com/w3c/wot-marketing/issues/216

### CODE REVIEW

**Comment by danielpeintner** [comment]
# Simplify navigation

Having just 3 entries

grafik


# Task-Forces

grafik


# Events

grafik


# Meetings

grafik

**Comment by egekorkan** [comment]
@mmccool this looks good to me but I am not 100% satisfied with the word activities. Do you have an alternative proposal?

File: docs/_config.yml [hunk 1]
```diff
@@ -18,22 +18,12 @@ navigation:
      url: cg/
  - title: Activities
    subnavigation:
-   - title: General
+   - title: Task Forces
      url: activities/general/
```
Comment (egekorkan) [inline docs/_config.yml:22]:
```suggestion
     url: activities/task-forces/
```

**Comment by egekorkan** [comment]
Thank you @danielpeintner ! Merging as agreed in today's call

**Pull request merged**